sql/database.sql

```diff
@@ -1620,33 +1620,37 @@ CREATE TABLE `standardized_tables_track` (
 
 DROP TABLE IF EXISTS `facility`;
 CREATE TABLE `facility` (
-  `id` int(11) NOT NULL auto_increment,
-  `name` varchar(255) default NULL,
-  `phone` varchar(30) default NULL,
-  `fax` varchar(30) default NULL,
-  `street` varchar(255) default NULL,
-  `city` varchar(255) default NULL,
-  `state` varchar(50) default NULL,
-  `postal_code` varchar(11) default NULL,
-  `country_code` varchar(30) NOT NULL default '',
-  `federal_ein` varchar(15) default NULL,
-  `website` varchar(255) default NULL,
-  `email` varchar(255) default NULL,
-  `service_location` tinyint(1) NOT NULL default '1',
-  `billing_location` tinyint(1) NOT NULL default '0',
-  `accepts_assignment` tinyint(1) NOT NULL default '0',
-  `pos_code` tinyint(4) default NULL,
-  `x12_sender_id` varchar(25) default NULL,
-  `attn` varchar(65) default NULL,
-  `domain_identifier` varchar(60) default NULL,
-  `facility_npi` varchar(15) default NULL,
-  `facility_taxonomy` varchar(15) default NULL,
-  `tax_id_type` VARCHAR(31) NOT NULL DEFAULT '',
-  `color` VARCHAR(7) NOT NULL DEFAULT '',
-  `primary_business_entity` INT(10) NOT NULL DEFAULT '0' COMMENT '0-Not Set as business entity 1-Set as business entity',
-  `facility_code` VARCHAR(31) default NULL,
+  `oid` int(15) NOT NULL DEFAULT '0' COMMET 'HIEs CCDA and FHIR an OID is required/wanted',
+  `id` int(11) NOT NULL,
```

Comment:
Why was the auto_increment removed?

Reply:
unintentional and it did not come out when I dumped the structure of the table.